src/store/modules/comments/actions/index.js

```diff
@@ -0,0 +1,111 @@
+import axios from 'axios';
+import * as Toastr from 'toastr';
+import { getSuccessResponse, getErrorResponse } from '@utils/getResponse';
+import getUserInfo from '../../../../utils/getUserFromToken';
+import {
+  GET_COMMENTS_PENDING,
+  GET_COMMENTS_SUCCESS,
+  GET_COMMENTS_FAILURE,
+  POST_COMMENT_PENDING,
+  POST_COMMENT_SUCCESS,
+  POST_COMMENT_FAILURE
+} from '../actionTypes';
+
+const removeQuotesFromString = str => str.trim().slice(1, str.length - 1);
+
+export const getCommentPending = () => ({
+  type: GET_COMMENTS_PENDING,
+  payload: {
+    status: 'commentPending',
```

Comment:
Useful nit-pick.

You could have used your ACTION_TYPES as the corresponding status as well. This gives the benefit of syntax highlighting and autocompletion. Also, helps eliminate bugs.